cdap-app-templates/cdap-program-report/cdap-program-report-base/src/main/java/co/cask/cdap/report/proto/ReportIdentifier.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.report.proto;
+
+/**
+ * ReportIdentifier represented by userName and reportId
+ */
+public class ReportIdentifier {
+    private final String userName;
```

Comment:
Wrong indentation. Should be 2 spaces.

Reply:
i am not sure why, but after rebasing to base spark module, my formatting settings aren't getting applied, i have formatted manually to resolve checkstyle.